library/core/src/num/uint_macros.rs

```diff
@@ -109,6 +109,10 @@ macro_rules! uint_impl {
 
         /// Returns the number of leading zeros in the binary representation of `self`.
         ///
+        /// The
+        #[doc = concat!("[`", stringify!($SelfTy), "::ilog2()`]")]
```

Comment:
Based on the CI failure, it looks like you don't need the `stringify!`:
```suggestion
        #[doc = concat!("[`", $SelfTy, "::ilog2()`]")]
```